config/redirects

```diff
@@ -6,3 +6,4 @@ raw: ${prefix}/ -> ${base}/current/
 raw: ${prefix}/stable -> ${base}/current/
 
 [*-master]: ${prefix}/${version}/fundamentals/versioned-api/ -> ${base}/${version}/fundamentals/stable-api/
+[v4.7-*]: ${prefix}/${version}/fundamentals/csfle -> ${base}/${version}/fundamentals/encrypt-fields
```

Comment:
Issue: there's currently no friendly docs 404 page (S3 NoSuchKey error), so we need to provide the reverse mapping for all other versions in case the reader switches versions while on the new page.

Suggestion:
```
[v3.6-v4.7): ${base}/${version}/fundamentals/encrypt-fields -> ${prefix}/${version}/fundamentals/csfle
```